### ISSUE TITLE
Nuget package for IQToolkit 2.0 

### ISSUE THREAD

**Issue by tb-mtg** [mattwar/iqtoolkit#22]
I cannot find the NuGet package for IQToolkit 2.0, can you please create one?

**Comment by Japollack** [comment]
Yes, what it takes to create a new NuGet package? Latest IQToolkit package is v1.0 from year 2017.
